bin/oneoff/nces_data/import_school_districts_2013_2014_ccd

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Is there any reason to leave these scripts around in case we need to repopulate the table from scratch someday? or would it be easy enough to rewrite these or use some other process to do that?

Reply:
I moved the logic that was in these files into the `seed_from_s3` method in the `school_district` model. So, if you started from scratch, you could run that method and populate the school districts table within 7 districts of what is in production currently (I tried to describe in the PR description a bit, let me know if I can clarify)